### PR TITLE
Dropout v2

### PR DESCRIPTION
**Pull request by finalelement**
Spleen Active Learning Dropout Segmentation App. This is a complete working version of the app. A review would be good from all reviewers.

### CODE REVIEW

**Comment by finalelement** [comment]
Active learning with Dropout has been integrated. Currently, we used already pre-trained models with dropout of 0.0. and the dropout is only used during inference time to generate uncertainty. The integration was performed in this [pull request](https://github.com/Project-MONAI/MONAILabel/pull/359). 

Hence closing off this one.
